modules/file-system/parallelstore/README.md

```diff
@@ -4,6 +4,11 @@ This module creates [parallelstore](https://cloud.google.com/parallelstore)
 instance. Parallelstore is Google Cloud's first party parallel file system
 service based on [Intel DAOS](https://docs.daos.io/v2.2/)
 
+### Supported Operating Systems
+
+The parallelstore instance created here can be used with Slurm cluster or compute
```

Comment:
s/The parallelstore instance created here/A parallelstore instance/

now that you are moving it out of the example, slight tweak to wording.